openedx/features/wikimedia_features/admin_dashboard/static/admin_dashboard/js/course_reports.js

```diff
@@ -0,0 +1,79 @@
+(function() {
+    'use strict';
+    var getCookie, ReportDownloads;
```

Comment:
Kindly try to use `let` in place of `var` as much as possible to keep the scope of variables limited.

Reply:
sure, taken note